docs/DeveloperGuide.md

```diff
@@ -1027,6 +1027,22 @@ Similar to **UC03 Delete a contact**, except,
 * a list of projects is shown instead of contacts.
 * a project is deleted instead of a contact.
 
+**Use case: UC24 Edit a project**
+
+Similar to **UC02 Edit a contact**, except, 
+* a list of projects is shown instead of contacts. 
+* a project's details is edited instead of a contact's details.
+
+**Use case: UC25 Removing a project's fields**
+
+Similar to **UC17 Removing tag/language(s)**, except, 
+* a project's fields is removed instead of tag/languages(s)
```

Comment:
Perhaps "fields are", I don't know how to make it modular for single or plural otherwise